src/lib/gas-refund.ts

```diff
@@ -17,6 +17,22 @@ export const GasRefundGenesisEpoch = 9;
 export const GasRefundPricingAlgoFlipEpoch = 11;
 export const GasRefundSafetyModuleStartEpoch = 11;
 export const GasRefundTxOriginCheckStartEpoch = 12;
+// todo: set this number accordingly
+export const GasRefundSwapSourceCovalentStartEpoch = 13;
+
+
+// todo: missing 2,5,6 - check that's expected
+export const STAKING_POOL_ADDRESSES: Record<string, string> = {
```

Comment:
No need for duplication. Can just import where needed and filter by active like done here https://github.com/paraswap/paraswap-volume-tracker/pull/34/files#diff-741f1ea4e96bea41c3cba7a9469511287af425ae3caebc62f4a418aa91e5bd49R27